### PR TITLE
Added opt for research button

### PR DESCRIPTION
**Pull request by yashlamba**

### CODE REVIEW

File: src/Home.js [hunk 1]
```diff
@@ -142,7 +145,7 @@ function Home(props) {
         </div>
       ) : (<> </>)}
       <div className="grid">
-        <form onSubmit={(e) => sendImage(e)}>
+        <form onSubmit={(e) => sendImage(e)} id="mainform">
```
Comment (yashlamba) [inline src/Home.js:148]:
```suggestion
        <form onSubmit={(e) => sendImage(e)}>
```

File: src/Home.js [hunk 2]
```diff
@@ -19,6 +19,7 @@ function Home(props) {
     5: "Fetched!",
     6: "Error"
   }
+  const url = "http://localhost:8000";
```
Comment (sakshamarora1) [inline src/Home.js:22]:
```suggestion
  const url = "http://handwrite-server.herokuapp.com";
```

**Review by sakshamarora1 — APPROVED** [review]
This is working fine. But we need to recheck the server code, as _1000 is being appended in both cases.

File: src/Home.js (hunk 2 above)
Comment (yashlamba) [reply src/Home.js:22]: Keeping it localhost in dev for now, we'll change it when deploying.

**Comment by yashlamba** [comment]
Fixed server issue here: https://github.com/cod-ed/handwrite-server/pull/13

**Pull request merged**